depends/patches/boost/deprecated-two-arg-allocate.diff

```diff
@@ -0,0 +1,102 @@
+diff -ur orig/include/boost/core/allocator_access.hpp patched/include/boost/core/allocator_access.hpp
```

Comment:
Is this patch reintroducing deprecated functionality, or backporting functionality that's needed to work around our current use of deprecated functionality?

Reply:
This patch eliminates the usage of deprecated functionality within Boost. AFAICT in Boost 1.75 Boost stopped relying on this functionality, but libc++ 11 starts requiring that it doesn't even exist in the codebase (or we get warnings, which break CI because we use `-Werror` on Linux).

Reply:
I had missed the fact that most of this is context lines, the github UI confused me. :+1: